src/de/cuuky/varo/configuration/configurations/language/languages/LanguageEN.java

```diff
@@ -135,6 +135,9 @@ public enum LanguageEN implements LoadableMessage {
 	NAMETAG_PREFIX_TEAM("nametag.suffix.noTeam", "%colorcode%%team% &7"),
 	NAMETAG_SUFFIX_TEAM("nametag.suffix.team", "&c %kills%"),
 
+    PLACEHOLDER_NO_TOP_PLAYER("placeholder.noTopPlayer", "Unknown"),
+    PLACEHOLDER_NO_TOP_TEAM("placeholder.noTopTeam", "Unknown"),
```

Comment:
```suggestion
    PLACEHOLDER_NO_TOP_PLAYER("placeholder.noTopPlayer", "-"),
    PLACEHOLDER_NO_TOP_TEAM("placeholder.noTopTeam", "-"),
```
I think we should keep "-" as the default value